hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -875,6 +879,90 @@ HapiSpec directTransferWorksForERC20() {
                                                         .withErcFungibleTransferStatus(true)))))));
     }
 
+    // test unprotected legacy ethereum transactions before EIP155
+    // this tests the behaviour when the `v` field is 27 or 28
```

Comment:
Can this comment on this test - also the next test - point to some documentation somewhere that explains what's going on here?  I.e., so you can know what the behavior is _in words_ before reading these tests and trying to reverse-engineer the desired behavior _from_ the tests just assuming they're correct and complete?